### DISCUSSION TITLE
Random Error "INTERNAL FUNCTION INVOCATION FAILED"

### DISCUSSION THREAD

**Discussion by lucasgerard** [vercel/vercel#6491] [Help] [answered]
Hello!

We're facing the following error on a random basis: `INTERNAL FUNCTION INVOCATION FAILED`.

This is not happening very frequently, but the problem is that once a mobile application got this error, it seems to be "stuck" on that error and the client will keep receiving this error, unless we do a factory-reset of the device.

This is very strange, and might be related to some DNS / cache issues, but we could not pin down the reason for now...

Here are logs from our log drain LogDNA and Sentry

Thank you for your help!

Lucas

![image](https://user-images.githubusercontent.com/5589858/126163560-5551d5b0-0b0f-42b5-abc0-bd15e0e2fc8a.png)

![image](https://user-images.githubusercontent.com/5589858/126164078-2a990583-73e5-4679-980b-5df835129094.png)

**Comment by leerob** [comment]
Hey! It's hard to tell the root cause based on the information you've provided. Could you try to get us a small, reproducible example?

**Comment by paulogdm** [comment, marked as answer]
Hello team,

This appears to be a Vercel bug :disappointed: Apologies for the inconvenience @lucasgerard .

If you see this bug, please send a message to support on https://vercel.com/support 

cc @leerob

**Comment by yachaka** [comment]
Hello Vercel team,

I'm Ilyes and I work with Shotgun on the mobile application that regularly receives this error.
Here is a visual that shows our received errors :
![invocation-failed](https://user-images.githubusercontent.com/8074336/132667099-f2ad9db9-19e2-4596-bd03-ec2dff7ed5c5.png)

This is an important issue because when a user is impacted, the user will keep being impacted (AKA can't log in to our service) for a big amount of time.

This is happening for months now, we did send many messages to your support email.

Can you at least explain to us what you see on your side so we can "go around" this issue with some tricks ?
The hard thing is also that the text response from the http function is nothing helpful to take action on our side ("function invocation failed").
If we knew where is this issue coming from, we could maybe find a way to get around it.

Please let me know if you have any more questions about the issue as I think I clearly shared with you all the details we have,
Thanks

**Reply by yachaka** [reply]
Sorry I had a miscalculation in the visual at the end,
It's more like 5 impacted users a day for us but it's very frustrating because once a user is impacted, he/she can't log in at all for a long time.

**Reply by yachaka** [reply]
@paulogdm
